src/main/cms/cms_menu_imu.c

```diff
@@ -391,6 +391,73 @@ static CMS_Menu cmsx_menuFilterPerProfile = {
     .entries = cmsx_menuFilterPerProfileEntries,
 };
 
+static uint8_t cmsx_dstPidProfile;
```

Comment:
OK, did not realize you where re-using conditionals from another PR. That is no good either, even worse. Each feature must have its own conditionals. We should not bundle and connect different things under general names like that. The names off conditionals should be meaningful and refer to the feature they control. Please change to something more unique, like `#ifdef USE_INTRA_PROFILE_COPY` or similar. And add it to all related code. It is not just a CMS feature.

PR #3601 should also change yes, but that is another case. Lets not mix that into this one...